### ISSUE TITLE
Use of jForms in Jelix 1.6.39 opt version results in PHP fatal error

### ISSUE THREAD

**Issue by zl4yy** [jelix/jelix#394]
Hi,

This seems to be a regression similar to [#380](https://github.com/jelix/jelix/issues/380).
I tried to manually copy this file https://raw.githubusercontent.com/jelix/jelix/jelix-1.6.x/lib/jelix/forms/controls/jFormsControlUpload2.class.php but for some reasons it does not work.
Note that the file seems to be in the "forms/controls" directory while the required_once calls for a file in "forms".

Cheers,
Yan.

**Comment by laurentj** [comment]
Thank you for the report. I fixed the bug and release Jelix 1.6.40. Note that this is the last release of Jelix 1.6. You should migrate your application to Jelix 1.7, or the new 1.8.0 which will be released today.

**Comment by zl4yy** [comment]
Hi Laurent.
Thanks for updating Jelix 1.6 for so many years. I know I have to transition to 1.7 but unfortunately my website was so "customised" that when I tried about 2 years ago I had issues with the URL engine and then stayed on 1.6. I'll try to upgrade again later this year.
By the way, Jelix 1.6.39 worked perfectly on PHP 8.1. The only part of my website where I had to spend time fixing poorly written code was the TCPDF class.
Thanks again for your work.
Regards,
Yan.
